src/main/java/datastore/Datastore.java

```diff
@@ -0,0 +1,52 @@
+package datastore;
+
+import com.google.auth.oauth2.GoogleCredentials;
+import com.google.cloud.firestore.CollectionReference;
+import com.google.cloud.firestore.Firestore;
+import com.google.firebase.FirebaseApp;
+import com.google.firebase.FirebaseOptions;
+import com.google.firebase.cloud.FirestoreClient;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+
+public class Datastore
+{
+    private static final String DATABASE_URL = "https://fitbook-746a5.us-east1.firebaseio.com/";
+    private final Firestore database;
+    private static Datastore instance;
```

Comment:
```suggestion
    private static volatile Datastore instance;
```
To ensure thread safety